### PR TITLE
#153728036 Add a feature to alert the user when a job runs …

### PR DESCRIPTION
**Pull request by Basemera**
…too often
**what does this PR do ?**
This PR notifies a user when a job runs too often.

**Description of task to be completed**
The user should be notified when a job runs too often.

**How should this be manually tested?**
Clone and run the app locally then run the tests using the following command:
or clone and run the web app by following the instruction indicated in the readme.md file.
Run the command ./manage.py sendalerts to run the server for sending alerts.
Run the ./manage.py runserver 8080 to start the server running.
Navigate to http://localhost:8080/checks/ in your browser, sign in and create your checks. Then you can ping them.

**What are the relevant pivotal tracker stories?**

[153728036](https://www.pivotaltracker.com/story/show/153728036)

### CODE REVIEW

File: hc/api/models.py [hunk 1]
```diff
@@ -100,6 +104,16 @@ def in_grace_period(self):
...
         return up_ends < timezone.now() < grace_ends
 
+#define reversed grace period
+    def reversed_grace_period(self):
+        if self.status in ("new", "paused"):
+            return False
+
+        up_ends = self.last_ping + self.timeout
+        reversed_grace_begins = up_ends - self.grace
+        # return up_ends > timezone.now() < reversed_grace_begins
```
Comment (jimmykimani) [inline hc/api/models.py]: I bet you don't need the commented code

Comment (Basemera) [reply hc/api/models.py]: Ya I will get rid of it

**Review by kaguna — COMMENTED** [review]
👍

**Review by mwaz — COMMENTED** [review]
👍

**Comment by anyric** [comment]
👍

**Review by anyric — APPROVED** [review]
👍 LGTM

**Pull request merged**